builder/base/cuda-10.2-jetson/build.sh

```diff
@@ -21,7 +21,8 @@ mkdir -p _jetson-rootfs
 sudo mount -o ro "${LOOP_DEV}p1" _jetson-rootfs
 sudo tar cf _jetson-rootfs.tar -C _jetson-rootfs .
 
-sudo docker run --rm --privileged multiarch/qemu-user-static --reset -p yes
-sudo docker build -t asia.gcr.io/pfn-public-ci/cupy-release-tools:cuda-10.2-jetson .
+[[ $(uname -m) = aarch64 ]] || docker run --rm --privileged multiarch/qemu-user-static --reset -p yes
```

Comment:
Note: this is to run `multiarch/qemu-user-static` if the node is not aarch64. e.g., to allow building aarch64 docker images on x86_64. On aarch64 node `multiarch/qemu-user-static` fails.